Sources/CartonKit/Server/Application.swift

```diff
@@ -50,7 +50,7 @@ extension Application {
     }
 
     webSocket("watcher") { request, ws in
-      let environment = request.headers["User-Agent"].compactMap(DestinationEnvironment.init).first
+      let environment = request.headers["User-Agent"].compactMap(DestinationEnvironment.from).first
```

Comment:
Could this be fixed without changing as much by specifying the argument label directly?
```suggestion
      let environment = request.headers["User-Agent"].compactMap(DestinationEnvironment.init(userAgent:)).first
```